api/Element.json

```diff
@@ -6593,10 +6593,15 @@
               "ie": {
                 "version_added": false
               },
-              "opera": {
-                "version_added": "48",
-                "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
-              },
+              "opera": [
+                {
+                  "version_added": "48",
+                  "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
+                },
+                {
+                  "version_added": true
```

Comment:
The data as proposed here suggests that

* This feature was supported in some version before 48
* From version 48 onward, the note applies

That doesn't really match up with your PR description. Are you're trying to say that, at some point after version 48, support for the `behavior` option was added too?